packages/block-library/src/rich-image/rich-image/index.js

```diff
@@ -210,57 +209,73 @@ class RichImage extends Component {
 
 				{ isEditing && (
 					<BlockControls>
-						<Toolbar>
-							<DropdownMenu
-								icon={ <RotateLeftIcon /> }
-								label={ __( 'Rotate' ) }
-								popoverProps={ POPOVER_PROPS }
-								controls={ [
-									{
-										icon: <RotateLeftIcon />,
-										title: __( 'Rotate left' ),
-										isDisabled: inProgress,
-										onClick: () =>
-											this.adjustImage( 'rotate', {
-												angle: -ROTATE_STEP,
-											} ),
-									},
-									{
-										icon: <RotateRightIcon />,
-										title: __( 'Rotate right' ),
-										isDisabled: inProgress,
-										onClick: () =>
-											this.adjustImage( 'rotate', {
-												angle: ROTATE_STEP,
-											} ),
-									},
-								] }
-							/>
-							<DropdownMenu
-								icon={ <FlipVerticalIcon /> }
-								label={ __( 'Flip' ) }
-								popoverProps={ POPOVER_PROPS }
-								controls={ [
-									{
-										icon: <FlipVerticalIcon />,
-										title: __( 'Flip vertical' ),
-										isDisabled: inProgress,
-										onClick: () =>
-											this.adjustImage( 'flip', {
-												direction: 'vertical',
-											} ),
-									},
-									{
-										icon: <FlipHorizontalIcon />,
-										title: __( 'Flip horizontal' ),
-										isDisabled: inProgress,
-										onClick: () =>
-											this.adjustImage( 'flip', {
-												direction: 'horizontal',
-											} ),
-									},
-								] }
-							/>
+						<ToolbarGroup>
+							<ToolbarItem>
+								{ ( toggleProps ) => (
+									<DropdownMenu
+										icon={ <RotateLeftIcon /> }
+										label={ __( 'Rotate' ) }
+										popoverProps={ POPOVER_PROPS }
+										toggleProps={ toggleProps }
+										controls={ [
+											{
+												icon: <RotateLeftIcon />,
+												title: __( 'Rotate left' ),
+												isDisabled: inProgress,
+												onClick: () =>
+													this.adjustImage(
+														'rotate',
+														{
+															angle: -ROTATE_STEP,
+														}
+													),
+											},
+											{
+												icon: <RotateRightIcon />,
+												title: __( 'Rotate right' ),
+												isDisabled: inProgress,
+												onClick: () =>
+													this.adjustImage(
+														'rotate',
+														{
+															angle: ROTATE_STEP,
+														}
+													),
+											},
+										] }
+									/>
+								) }
+							</ToolbarItem>
```

Comment:
The diff got messed up. But this is basically wrapping `<DropdownMenu>` with `<ToolbarItem>` and passing the toolbar item props to `toggleProps`:
```jsx
<ToolbarItem>
	{ ( toggleProps ) => (
		<DropdownMenu
			...
			toggleProps={ toggleProps }
		/>
	) }
</ToolbarItem>
```